### PR TITLE
Fixes #23505: Add sales-tier data to release-data

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/23505

### CODE REVIEW

File: release-data/rudder-7.0.cson [hunk 1]
```diff
@@ -88,6 +88,37 @@ tier1 = [
...
+    "debian-9":       { "i386": "tier2", "amd64": "tier2", "armhf": "tier2", "arm64": "tier2" },
+    "debian-10":      { "i386": "tier1", "amd64": "tier1", "armhf": "tier2", "arm64": "tier2" },
+    "debian-11":      { "i386": "tier1", "amd64": "tier1", "armhf": "tier2", "arm64": "tier2" },
+    "ubuntu-10.04":   { "i386": "tier3", "amd64": "tier3" },
+    "ubuntu-12.04":   { "i386": "tier3", "amd64": "tier3" },
+    "ubuntu-13.04":   { "i386": "tier3", "amd64": "tier3" },
+    "ubuntu-14.04":   { "i386": "tier3", "amd64": "tier3" },
+    "ubuntu-15.10":   { "i386": "tier3", "amd64": "tier3" },
+    "ubuntu-16.04":   { "i386": "tier2", "amd64": "tier2" },
+    "ubuntu-18.04":   { "amd64": "tier1" },
```
Comment (ncharles) [inline release-data/rudder-7.0.cson:105]: don't we have a i386 tiers2 ?

File: release-data/rudder-7.0.cson [hunk 2]
```diff
@@ -88,6 +88,37 @@ tier1 = [
...
+    "ubuntu-20.04":   { "amd64": "tier1" },
+    "rhel-5":         { "i386": "tier4", "amd64": "tier4" },
```
Comment (ncharles) [inline release-data/rudder-7.0.cson:107]: I don't have that in the spreadsheet, it should be
"rhel-3":         { "i386": "tier4", "amd64": "tier4" },
"rhel-5":         { "i386": "tier3", "amd64": "tier3" },

File: release-data/rudder-7.0.cson [hunk 3]
```diff
@@ -88,6 +88,37 @@ tier1 = [
...
+    "rhel-6":         { "i386": "tier3", "amd64": "tier3" },
+    "rhel-7":         { "i386": "tier3", "amd64": "tier1", "ppc64le": "tier1" },
+    "rhel-8":         { "amd64": "tier1", "ppc64le": "tier1" },
+    "sles-11"   :     { "i386": "tier2", "amd64": "tier2" },
```
Comment (ncharles) [inline release-data/rudder-7.0.cson:111]: we should have
 "sles-10"   :     { "i386": "tier4", "amd64": "tier4" },

File: release-data/rudder-7.1.cson [hunk 4]
```diff
@@ -89,6 +89,37 @@ tier1 = [
           "slackware-14.1", "slackware-14.2",
         ]
 
+# see https://docs.google.com/spreadsheets/d/1MbpYDH5Eqmgj1j-G4Gu0oKQoBaiGhjUfwVY5qtXTsu4/edit#gid=0
+sales-tier = {
```
Comment (ncharles) [inline release-data/rudder-7.1.cson:93]: i'm not commenting here, the same as 7.0 should apply (and extra os added during 7.1)

File: release-data/rudder-7.2.cson [hunk 5]
```diff
@@ -91,6 +91,39 @@ tier1 = [
           "slackware-14.1", "slackware-14.2",
         ]
 
+# see https://docs.google.com/spreadsheets/d/1MbpYDH5Eqmgj1j-G4Gu0oKQoBaiGhjUfwVY5qtXTsu4/edit#gid=0
+sales-tier = {
```
Comment (ncharles) [inline release-data/rudder-7.2.cson:95]: same here

File: release-data/rudder-8.0.cson [hunk 6]
```diff
@@ -91,6 +91,40 @@ tier1 = [
           "slackware-14.1", "slackware-14.2", "slackware-15.0"
         ]
 
+# see https://docs.google.com/spreadsheets/d/1MbpYDH5Eqmgj1j-G4Gu0oKQoBaiGhjUfwVY5qtXTsu4/edit#gid=0
+sales-tier = {
+    "debian-6":       { "i386": "tier3", "amd64": "tier3" },
+    "debian-7":       { "i386": "tier3", "amd64": "tier3" },
```
Comment (ncharles) [inline release-data/rudder-8.0.cson:97]: debian 6 & 7 are both tiers4

File: release-data/rudder-8.0.cson [hunk 7]
```diff
@@ -91,6 +91,40 @@ tier1 = [
...
+    "debian-8":       { "i386": "tier3", "amd64": "tier3", "armhf": "tier3", "arm64": "tier3" },
```
Comment (ncharles) [inline release-data/rudder-8.0.cson:98]: debian 8 arm is tiers4

File: release-data/rudder-8.0.cson [hunk 8]
```diff
@@ -91,6 +91,40 @@ tier1 = [
...
+    "debian-9":       { "i386": "tier2", "amd64": "tier2", "armhf": "tier2", "arm64": "tier2" },
```
Comment (ncharles) [inline release-data/rudder-8.0.cson:99]: debian 9 arm is tiers4, and i386amd64 is tiers3

File: release-data/rudder-8.0.cson [hunk 9]
```diff
@@ -91,6 +91,40 @@ tier1 = [
...
+    "ubuntu-14.04":   { "i386": "tier3", "amd64": "tier3" },
+    "ubuntu-15.10":   { "i386": "tier4", "amd64": "tier4" },
+    "ubuntu-16.04":   { "i386": "tier3", "amd64": "tier3" },
+    "ubuntu-18.04":   { "i386": "tier2", "amd64": "tier2" },
+    "ubuntu-20.04":   { "amd64": "tier1" },
+    "ubuntu-22.04":   { "amd64": "tier1" },
+    "ubuntu-24.04":   { "amd64": "tier1" },
+    "rhel-6":         { "i386": "tier3", "amd64": "tier3" },
+    "rhel-7":         { "i386": "tier3", "amd64": "tier2", "ppc64le": "tier3" },
+    "rhel-8":         { "amd64": "tier1", "ppc64le": "tier2" },
```
Comment (ncharles) [inline release-data/rudder-8.0.cson:115]: ppc64 for rhel8 & 9 is tier3

File: release-data/rudder-8.0.cson [hunk 10]
```diff
@@ -91,6 +91,40 @@ tier1 = [
...
+    "ubuntu-12.04":   { "i386": "tier4", "amd64": "tier4" },
+    "ubuntu-13.04":   { "i386": "tier4", "amd64": "tier4" },
+    "ubuntu-14.04":   { "i386": "tier3", "amd64": "tier3" },
+    "ubuntu-15.10":   { "i386": "tier4", "amd64": "tier4" },
+    "ubuntu-16.04":   { "i386": "tier3", "amd64": "tier3" },
+    "ubuntu-18.04":   { "i386": "tier2", "amd64": "tier2" },
+    "ubuntu-20.04":   { "amd64": "tier1" },
+    "ubuntu-22.04":   { "amd64": "tier1" },
+    "ubuntu-24.04":   { "amd64": "tier1" },
+    "rhel-6":         { "i386": "tier3", "amd64": "tier3" },
```
Comment (ncharles) [inline release-data/rudder-8.0.cson:113]: where is rhel5 ?

File: release-data/rudder-8.0.cson [hunk 11]
```diff
@@ -91,6 +91,40 @@ tier1 = [
...
+    "rhel-7":         { "i386": "tier3", "amd64": "tier2", "ppc64le": "tier3" },
+    "rhel-8":         { "amd64": "tier1", "ppc64le": "tier2" },
+    "rhel-9":         { "amd64": "tier1", "ppc64le": "tier2" },
+    "al-2023":        { "amd64": "tier1" },
```
Comment (ncharles) [inline release-data/rudder-8.0.cson:117]: where is AL2?

File: release-data/rudder-8.0.cson [hunk 12]
```diff
@@ -91,6 +91,40 @@ tier1 = [
...
+    "sles-11"   :     { "i386": "tier3", "amd64": "tier3" },
```
Comment (ncharles) [inline release-data/rudder-8.0.cson:118]: we should have a sles10 tiers4 also

File: release-data/rudder-7.0.cson (hunk 1 above)
Comment (peckpeck) [reply release-data/rudder-7.0.cson:105]: we don't build it

File: release-data/rudder-8.0.cson (hunk 10 above)
Comment (peckpeck) [reply release-data/rudder-8.0.cson:113]: we stopped building it

**Comment by peckpeck** [comment]
PR updated with a new commit

**Review by ncharles — APPROVED** [review]

**Pull request merged**